dsl/promote.groovy

```diff
@@ -0,0 +1,24 @@
+import groovy.transform.BaseScript
+import com.electriccloud.commander.dsl.util.BasePlugin
+
+//noinspection GroovyUnusedAssignment
+@BaseScript BasePlugin baseScript
+
+// Variables available for use in DSL code
+def pluginName = args.pluginName
+def pluginKey = getProject("/plugins/$pluginName/project").pluginKey
+def pluginDir = getProperty("/server/settings/pluginsDirectory").value + "/" + pluginName
+
+project pluginName, {
+
+	description = 'ElectricFlow integration with DaticalDB.'
+	ec_visibility = 'pickListOnly'
+
+	property 'version', value = '1.1.0'
```

Comment:
this is a left over for when the plugin was created with EC-Admin. That property is used to create the plugin version.
I will remove it from the DSL as it's not necessary anymore and confusing obviously